keyboards/handwired/phantagom/gigan/keymaps/default/keymap.c

```diff
@@ -0,0 +1,261 @@
+        0,  0,  0,  0,  0,  3,  7,  3,  3,  3,  3,  7,  7,  3,  7, 15,  7,  7, 31, 15, 15, 15, 31, 31, 31,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,243,192,128,192,192,192,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,128,254, 63,  0,  0,255,255,  0,  3, 31,152,128,128,192,195,231,110, 60, 56, 60,126,231,195,193,128,128,128,  0,  0,  0,  0,255,255,219,153,153,153,255,255,153,153,153,255,255,  0,  0, 24, 24, 24, 24,255,255,220, 24, 24, 24, 25,  0,  0,  0,  0,  0,  0,  0, 
+        0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 24,  8,  8, 12,  4,  6,  6,  3,131,195,227,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,240,224,224, 96,  0,  0,  0,  0,  0,  0,  0,  0,  1,  0,  0,  0,255,255,  0,  0,  1,  3, 49, 49, 48, 48, 48, 48,254,254,254, 48, 48, 48, 48, 49, 49,  1,  0,  0,  0, 12, 12,236,236,108,108,108,108,108,108,108,108,236,236, 12,  4,  0,128,224,254, 15,  3, 31,248,192,  0,  0,  0,  0,  0,  0,  0,  0,  0, 
+        0,  0,  0,  0,  0,  0,  0,  0,248,248,240,240,248,254,254,252,255,254,254,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,254,252,252,128,  0,  0,  0,  0,  0,  0,  0,  0,  0, 63, 63,  0,  0, 24, 24, 24, 24, 24, 24, 24, 24, 31, 31, 31, 24, 24, 24, 24, 24, 24, 24, 24,  0,  0,  0,  0, 63, 63, 12, 12, 12, 12, 12, 12, 12, 12, 63, 63,  0, 24, 60, 15,  3,  0,  0,  0,  0,  1,  3, 14, 60, 16,  0,  0,  0,  0,  0,  0, 
+        0,  0,  0,  0,  0,  0,  0,  0,  0,  1,  1,  1,  0,  0,  0,  0,  0,  0,  0,  1,  1,  1,  1,  3,  7, 31,127,127,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,127,127,127, 63, 63, 63, 63, 31, 31, 31, 31, 30,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0, 
+        0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  3,  7,  7,  7,  7,  7,  7,  3,  3,  3,  3,  1,  1,  1,  1,  1,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,  0,
+     };
+    oled_write_raw((char*)raw_logo, sizeof(raw_logo));
+
+}
+
+// This is a variable to keep track of the time elapsed since power up
+static uint32_t logo_timer = 0;
+
+// Write all to oled
+bool oled_task_user(void) {
```

Comment:
This should be moved to the keyboard's c file, and using the `oled_task_kb` function, so that there is a defined default behavior, especially since this is being duplicated in both the default and via keymaps.